python/ray/util/client/common.py

```diff
@@ -64,6 +64,8 @@
 # Long timeout because we do not want gRPC ending a connection.
 GRPC_KEEPALIVE_TIMEOUT_MS = 1000 * 600
 
+CLIENT_SERVER_MAX_THREADS = float(os.getenv("RAY_CLIENT_SERVER_MAX_THREADS", 100))
```

Comment:
hmm why not int instead of float?

Reply:
I move it from L80 -> L67... So I'm not sure.